src/olympia/api/models.py

```diff
@@ -23,7 +23,7 @@ class APIKey(ModelBase):
     A developer's key/secret pair to access the API.
     """
     user = models.ForeignKey(UserProfile, related_name='api_keys')
-    is_active = models.BooleanField(default=True)
+    is_active = models.NullBooleanField(default=True)
```

Comment:
I had to read through the whole patch to realise _why_ `BooleanField` was changed to `NullBooleanField` (the constraint) - can you add a comment here about it?

Reply:
Sure, done in dcfe291086ce72fc9e5a1affe92a24453223c5e1